Feature/kas 2580 brondocument opladen

TaneeG opened the pull request:
Zie ook PR in project

Commit:
KAS-2580 Model aanpassingen + Toevoegen nieuwe routes

Commit:
Merge branch 'development' into feature/KAS_2580_brondocument_opladen

# Conflicts:
#	app/components/publications/publication/publication-navigation.hbs

Commit:
Merge branch 'development' into feature/KAS_2580_brondocument_opladen

Commit:
KAS-2580 First setup of translations templates

Commit:
Merge branch 'development' into feature/KAS_2580_brondocument_opladen

Commit:
KAS-2580 Uploading modal + table

Commit:
Merge branch 'development' into feature/KAS_2580_brondocument_opladen

Commit:
KAS-2580 commenting things out od scope and adding lannguage to pieces and changing piece textarea to input

Commit:
Merge branch 'development' into feature/KAS_2580_brondocument_opladen

Commit:
Merge branch 'development' into feature/KAS_2580_brondocument_opladen

Commit:
KAS-2474 fixing some unused code

erikap committed:
Merge branch 'development' into feature/KAS_2580_brondocument_opladen

erikap commented:
Backend PR: https://github.com/kanselarij-vlaanderen/kaleidos-project/pull/196/files

erikap committed:
Remove unnecessary async from hooks in route

erikap committed:
Fix classnames of translations routes and controllers

erikap committed:
Remove right margin on publication tabs

As long as the progress badges for each subcase aren't shown on the
tab, the right margin should not be there

erikap committed:
Refactor checkbox, buttond and empty state components to auk-components

erikap committed:
Apply style like prototype on translation tabs toolbar

erikap committed:
Add temporary styling for the buttons on the translation documents tab

erikap committed:
Use open- and close- action names for piece upload modal

erikap committed:
Add identification number to translation documents upload modal

erikap committed:
Refactor AuButton to Auk::Button

erikap committed:
Refactor AuInput to Auk::Input component

erikap committed:
Make amount of words and pages optional on document upload

erikap committed:
Fix translation for number of pages

erikap committed:
Add KAS-ticket references to commented regions

erikap committed:
Remove undefined default access level from translation document upload

erikap committed:
Set document name to file name by default for translation document

erikap committed:
Make translation subcase a tracked property no the translation
documents controller

erikap committed:
Fix capitalization of table headers

erikap committed:
Use named download link to download a file

erikap committed:
Fix linting

erikap reviewed (approved):
Meeste commits die ik nog toegevoegd heb zijn refactors van de Auk-components vanuit de development branch en style changes om in lijn te zijn met prototype.

MikiDi committed:
Merge branch 'development' into feature/KAS_2580_brondocument_opladen

MikiDi commented:
Nogmaals testen gestart @ http://kal-kastaar.s.redpencil.io:8080/job/kaleidos/job/custom_frontend_backend_combination/53/

Commit:
KAS-2580 Added new components + transistion to

ValenberghsSven commented:
hmmm, jenkins doet hier wat raars mee.
Er staan een 6-tal screenshots te zien, maar als je de console output bekijkt is er maar 1 test gefaald

19:58:54      ✖  1 of 44 failed (2%)                    1:29:53      164      161        1        2        -  

mogelijk zit het script van die custom build nog niet helemaal juist qua opkuis/nieuwe screenshot folder bij builden van dezelfde combinatie waardoor we foute screenshots te zien krijgen

Ik fix die test nog even (css issue) en dan loop ik een nieuwe
Edit: was geen css issue, maar flakyness dat er soms errors zijn vlak na POST van /cases

ValenberghsSven commented:
de test dat op jenkins gefaald was werkt lokaal
![image](https://user-images.githubusercontent.com/22245223/121158847-f6e0e100-c84a-11eb-9dcb-04b8c6b93313.png)

ValenberghsSven commented:
voor de zekerheid
http://kal-kastaar.s.redpencil.io:8080/job/kaleidos/job/custom_frontend_backend_combination/54/

Deze is groen :heavy_check_mark:

Commit:
Merge branch 'development' into feature/KAS_2580_brondocument_opladen
